### ISSUE TITLE
Create form that prompts physician for all information needed for a patient to pass Diabetes Group Measure

### ISSUE THREAD

**Issue by leebc** [aethelwulffe/PQRS_Gateway#22]
Create a form that prompts the physician for all information needed for a patient to pass Diabetes Group Measure. Form creates entries in the database for the codes needed to pass the associated Numerator.

Measures and associated codes (logic copied from the measures) required to pass the numerator:
* Diabetes_0001 -- CPT2 code: 3046F
* Diabetes_0110 -- HicPic : 'G8482','G8483'
* Diabetes_0117 -- b1.code IN ( '2022F','2024F','2026F','3072F') AND b1.modifier != '8P'; ";
* Diabetes_0119 -- b1.code IN ( '3060F','3061F','3062F','3066F','G8506') AND b1.modifier != '8P'; ";
* Diabetes_0163 -- b1.code = 'G9226'
* Diabetes_0226 -- ( (b1.code = '4004F' AND b1.modifier !='8P') OR b1.code = '1036F')

**Comment by leebc** [comment]
* Optional Intent Code G8485: I intend to report the Diabetes Measures Group
* Composite QDC G8494 has been created for registries that utilize claims data. This QDC may be reported in lieu of individual QDCs when all quality clinical actions for all applicable measures within the group have been performed.  
-->  G8494: All quality actions for the applicable measures in the Diabetes Measures Group
have been performed for this patient

**Comment by leebc** [comment]
Measure #1 (NQF 0059): Diabetes: Hemoglobin A1c Poor Control -- National Quality Strategy
Domain: Effective Clinical Care
Data asked:  
* Most recent hemoglobin A1c level 
Values:
* Most recent hemoglobin A1c level > 9.0% (3046F)  (Performance Met, INVERSE MEASURE)
* Hemoglobin A1c level was not performed during the measurement period (12 months) (3046F with 8P)  (Performance Met, INVERSE MEASURE) 
* Most recent hemoglobin A1c (HbA1c) level < 7.0%   (3044F) (Performance Not Met, INVERSE MEASURE)
* Most recent hemoglobin A1c (HbA1c) level 7.0 to 9.0% (3045F) (Performance Not Met, INVERSE MEASURE)

**Comment by leebc** [comment]
Measure #110 (NQF 0041): Preventive Care and Screening: Influenza Immunization -- National
Quality Strategy Domain: Community/Population Health

Question asked: 
* Did patient receive an influenza immunization for this flu season OR report a previous receipt of an influenza immunization

Values:
* Influenza immunization administered or previously received (G8482) (Performance Met)
* Patient reports Influenza immunization previously received (G8482) (Performance Met) (same)
* Influenza immunization not administered, document reason (e.g., patient allergy or
other medical reasons, patient declined or other patient reasons, vaccine not available or other system reasons) (G8483) (Performance Met)
* Influenza immunization was not administered, reason not given (G8484) (Performance Not Met)

**Comment by leebc** [comment]
Measure #117 (NQF 0055): Diabetes: Eye Exam -- National Quality Strategy Domain: Effective
Clinical Care

Question asked: 

* Did the patient receive an eye screening for diabetic retinal disease this year?  (A retinal or dilated eye exam by an eye care professional this year or had no evidence of retinopathy in an eye exam in the previous year.)

Values:
* Dilated retinal eye exam with interpretation by an ophthalmologist or optometrist documented and reviewed (2022F) (Performance Met)
* Seven standard field stereoscopic photos with interpretation by an ophthalmologist or optometrist documented and reviewed (2024F)  (Performance Met)
* Eye imaging validated to match diagnosis from seven standard field stereoscopic photos results documented and reviewed (2026F)  (Performance Met)
* Low risk for retinopathy (no evidence of retinopathy in the prior year) (3072F)* (Performance Met)  (*NOTE: This code can only be used if the encounter was during the measurement period because it indicates that the patient had "no evidence of retinopathy in the prior year". This code definition indicates results were negative, therefore a result is not required.)
* Dilated eye exam was not performed, reason not otherwise specified (2022F or 2024F or 2026F with 8P)

**Comment by meggerc** [comment]
I created a spreadsheet listed the PQRS#, the CPTII and HCPC codes, and explanation. I'm unable to upload it because it's an .ods file and GitHub doesn't support ods.  I'm trying to figure out how to zip an ODS file.

**Comment by leebc** [comment]
Measure #119 (NQF 0062): Diabetes: Medical Attention for Nephropathy -- National Quality
Strategy Domain: Effective Clinical Care

Question asked: 
* Did the patient receive a nephropathy screening test or have evidence of nephropathy?

Values:
* Positive microalbuminuria test result documented and reviewed (3060F) (Performance Met)
* Negative microalbuminuria test result documented and reviewed (3061F) (Performance Met)
* Positive macroalbuminuria test result documented and reviewed (3062F) (Performance Met)
* Documentation of treatment for nephropathy (eg, patient receiving dialysis, patient being treated for ESRD, CRF, ARF, or renal insufficiency, any visit to a nephrologist) (3066F) (Performance Met)
* Patient receiving angiotensin converting enzyme (ACE) inhibitor or angiotensin receptor blocker (ARB) therapy (G8506) (Performance Met)
* Nephropathy screening was not performed, reason not otherwise specified (3060F or 3061F or 3062F with 8P)  (Performance Not Met)

**Comment by teryhill** [comment]
Have you tried right clicking on the file and selecting "Send to" and then compressed file

**Comment by leebc** [comment]
@meggerc : I'm pulling together all the specific pieces for the Diabetes Group in one place and writing up how I want to treat each piece.

**Comment by aethelwulffe** [comment]
It's a linux thing.

**Comment by meggerc** [comment]
@teryhill Thanks - that worked. 
[PQRS_2016_Diabetes_Group_Numerator_Codes.zip](https://github.com/aethelwulffe/PQRS_Gateway/files/235433/PQRS_2016_Diabetes_Group_Numerator_Codes.zip)

**Comment by leebc** [comment]
Measure #126 (NQF 0417): Diabetes Mellitus: Diabetic Foot and Ankle Care, Peripheral
Neuropathy â Neurological Evaluation -- National Quality Strategy Domain: Effective Clinical Care

Question Asked: 
Did the patient have a lower extremity neurological exam performed at least once within the last 12 months?

Values:
* Lower extremity neurological exam performed and documented (G8404) (Performance Met)
* Lower extremity neurological exam not performed(G8405) (Performance Not Met)

**Comment by leebc** [comment]
Measure #226 (NQF 0028): Preventive Care and Screening: Tobacco Use: Screening and
Cessation Intervention -- National Quality Strategy Domain: Community/Population Health

Question Asked:
* Was patient screened for tobacco use at least once within  24 months AND,  if identified as a tobacco user, did they receive tobacco cessation intervention if identified as a tobacco user?

Values:
* Current tobacco non-user (1036F)  (Performance Met)
* Patient screened for tobacco use AND received tobacco cessation intervention (counseling, pharmacotherapy, or both), if identified as a tobacco user (4004F)  (Performance Met) 
* Documentation of medical reason(s) for not screening for tobacco use (eg, limited life expectancy, other medical reasons) (4004F with 1P) (Exclusion)
* Tobacco screening OR tobacco cessation intervention not performed, reason not otherwise specified
(4004F with 8P)  (Performance Not Met)

**Comment by aethelwulffe** [comment]
Yeah, this thing LeeBC is writing up is his documentation for an actual 
diabetes treatment note form.
I am doing Sinusitis Group.  It is a bit simpler.  I will publish it in 
a few hours.
